clang-tools-extra/docs/clang-tidy/checks/boost/use-ranges.rst

```diff
@@ -0,0 +1,24 @@
+.. title:: clang-tidy - boost-use-ranges
+
```

Comment:
add options that check has (like include style).

Add table with info what is converted to what.